src/MultipleChoice/MultipleChoice.scss

```diff
@@ -17,6 +17,7 @@
   @include button-quiz-answer;
   align-self: center;
   margin: 2%;
+  width: 120px;
```

Comment:
Gender typical buttons are overflowing a bit on small screens (I tried on an iphone X)- they might need to be slightly smaller in general  @saraho1123